packages/database/prisma/migrations/20231026095637_add_path_exclusion_table/migration.sql

```diff
@@ -0,0 +1,19 @@
+-- CreateTable
```

Comment:
I believe backend should verify the path exclude reason is one of these:

enum class PathExcludeReason {
    /**
     * The path only contains tools used for building source code which are not included in distributed build
     * artifacts.
     */
    BUILD_TOOL_OF,

    /**
     * The path only contains data files such as fonts or images which are not included in distributed build
     * artifacts.
     */
    DATA_FILE_OF,

    /**
     * The path only contains documentation which is not included in distributed build artifacts.
     */
    DOCUMENTATION_OF,

    /**
     * The path only contains source code examples which are not included in distributed build artifacts.
     */
    EXAMPLE_OF,

    /**
     * The path only contains optional components for the code that is built which are not included in distributed
     * build artifacts.
     */
    OPTIONAL_COMPONENT_OF,

    /**
     * Any other reason which cannot be represented by any other element of [PathExcludeReason].
     */
    OTHER,

    /**
     * The path only contains packages or sources for packages that have to be provided by the user of distributed build
     * artifacts.
     */
    PROVIDED_BY,

    /**
     * The path only contains files used for testing source code which are not included in distributed build
     * artifacts.
     */
    TEST_OF,

    /**
     * The path only contains tools used for testing source code which are not included in distributed build
     * artifacts.
     */
    TEST_TOOL_OF
}

Reply:
Yes, true

Reply:
Will implement this now

Reply:
This is now  added